packages/rest/src/sample.ts

```diff
@@ -0,0 +1,27 @@
+import { LogLevel, AutoAcceptCredential } from '@aries-framework/core'
```

Comment:
Normally we place samples outside of the `src` directory in a separate `samples` directory